libs/vm_modules/include/vm_modules/math/math.hpp

```diff
@@ -35,6 +37,36 @@ fetch::math::meta::IfIsMath<T, T> Abs(fetch::vm::VM *, T const &a)
   return x;
 }
 
+template <typename T, typename R = void>
+using IfIsNormalSignedInteger = meta::EnableIf<meta::IsSignedInteger<T> && sizeof(T) >= 4, R>;
+
+template <typename T, typename R = void>
+using IfIsSmallSignedInteger = meta::EnableIf<meta::IsSignedInteger<T> && sizeof(T) < 4, R>;
```

Comment:
move both of these to `meta`?

Reply:
@ejfitzgerald

Reply:
Yes ultimately there needs to be some clean up with the abs. I have made it @baykaner 's problem